cluster/images/hyperkube/BUILD

```diff
@@ -1,16 +1,42 @@
+load("@io_bazel_rules_docker//container:container.bzl", "container_layer")
 load("//build:container.bzl", "multi_arch_container")
 load("//build:platforms.bzl", "SERVER_PLATFORMS")
 
+container_layer(
+    name = "scripts",
+    directory = "/",
+    files = [
+        "hyperkube",
+    ],
+)
+
+container_layer(
+    name = "bins",
+    directory = "/usr/local/bin",
+    files = [
+        "//cmd/kube-apiserver",
+        "//cmd/kube-controller-manager",
+        "//cmd/kube-proxy",
+        "//cmd/kube-scheduler",
+        "//cmd/kubectl",
+        "//cmd/kubelet",
+    ],
+)
+
 multi_arch_container(
-    name = "hyperkube",
+    name = "image",
```

Comment:
was there a reason this changed names? anyone using the existing build target would have to change their scripts, right?

Reply:
the name was conflicting with the name of the script. if you see the conformance image, we use `"image"`, so changed this to match. 

note that this is not used by release automation and i could not find any references anywhere else, so we should be fine.

Reply:
ok

Reply:
we use "image" in kubemark for example https://github.com/kubernetes/kubernetes/blob/master/cluster/images/kubemark/BUILD#L6